### PR TITLE
Change stats styles

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

File: src/components/DebtPool/DebtPool.tsx [hunk 1]
```diff
@@ -43,25 +55,15 @@ export const DebtPool: React.FC<IProps> = ({ title, subTitle, data }) => {
                 arcLabelsRadiusOffset={0}
                 colors={{ datum: 'data.color' }}
                 arcLabelsTextColor='#000000'
-                tooltip={({ datum: { color, label } }) => (
+                tooltip={() => (
```
Comment (awojciak) [inline src/components/DebtPool/DebtPool.tsx:58]: I think, null as value of this prop should be enough

File: src/components/DebtPool/style.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontSize: '22px',
     lineHeight: '22px',
     fontWeight: 600,
-    paddingBottom: "8px"
+    paddingBottom: '8px'
```
Comment (Kimi82) [inline src/components/DebtPool/style.ts:24]: You don't have to type 'px' in padding.
Only lineHeight require it

**Review by Kimi82 — APPROVED** [review]

File: src/components/DebtPool/DebtPool.tsx [hunk 3]
```diff
@@ -70,6 +86,9 @@ export const DebtPool: React.FC<IProps> = ({ title, subTitle, data }) => {
                 }}
                 onMouseLeave={event => {
                   const variable: string = event.id.toString()
+                  setLabel('Total Dept')
```
Comment (awojciak) [inline src/components/DebtPool/DebtPool.tsx:89]: it should be "Debt"

Comment (awojciak) [reply src/components/DebtPool/DebtPool.tsx:89]: and you should remove this box-shadow: 
obraz

Comment (awojciak) [reply src/components/DebtPool/DebtPool.tsx:89]: and I think that the amount should be bold, maybe ask Agnieszka about that

File: src/components/DebtPool/DebtPool.tsx [hunk 4]
```diff
@@ -18,23 +18,48 @@ export interface IProps {
...
+  const [percent, setPercent] = React.useState<String>('')
   const classes = useStyles()
+  const synthSumValue = (): number => {
+    let sum = 0
+    data.map(item => {
+      sum += item.price
+    })
+    return sum
+  }
+  const synthSum = synthSumValue() as Number
```
Comment (awojciak) [inline src/components/DebtPool/DebtPool.tsx:32]: does this value change when values in data change?

**Review by awojciak — CHANGES_REQUESTED** [review]
look at my comments

File: src/components/DebtPool/DebtPool.tsx (hunk 1 above)
Comment (PawelKrawczyk96) [reply src/components/DebtPool/DebtPool.tsx:71]: tooltip prop must returned RC, so i add a disabled div. If i add null as value TS will throw an error.

Comment (awojciak) [reply src/components/DebtPool/DebtPool.tsx:70]: maybe just function returning null? something like that is used in collateral plot, check it

**Review by wojciech-cichocki — APPROVED** [review]
Maybe it will be a good idea to add similar effect also to token list

File: src/containers/DebtPoolContainer/DebtPoolContainer.tsx [hunk 5]
```diff
@@ -38,7 +38,7 @@ export const DebtPoolContainer: React.FC<IProps> = () => {
       <Grid className={classes.debtcontainer}>
         <DebtPool
           title='Debt pool'
-          subTitle='Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+          subTitle='Percentage graph of a given synthetics share in debt'
```
Comment (awojciak) [inline src/containers/DebtPoolContainer/DebtPoolContainer.tsx:41]: maybe "Chart of percentage share in total debt of available synthetic assets"?

File: src/components/Stats/StatsCollateralChart/StatsCollateral.tsx [hunk 6]
```diff
@@ -67,7 +67,7 @@ export const StatsCollateralChart: React.FC<IProps> = ({ data }) => {
         <Grid container className={classes.headerWrapper} direction="column">
           <Grid item>
             <Typography className={classes.title}>Collateral structure</Typography>
-            <Typography className={classes.subTitle}>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</Typography>
+            <Typography className={classes.subTitle}>Percentage graph of the amount of collaterals</Typography>
```
Comment (awojciak) [inline src/components/Stats/StatsCollateralChart/StatsCollateral.tsx:70]: maybe "Chart of percentage share of total deposit of available collaterals"?

Comment (awojciak) [reply src/components/Stats/StatsCollateralChart/StatsCollateral.tsx:70]: no, "Chart of total deposit's percentage share for each available collateral"

File: src/containers/DebtPoolContainer/DebtPoolContainer.tsx (hunk 5 above)
Comment (awojciak) [reply src/containers/DebtPoolContainer/DebtPoolContainer.tsx:41]: no, "Chart of total debt's percentage share for each available synthetic asset"

**Review by awojciak — APPROVED** [review]

**Pull request merged**